WordPress/src/main/java/org/wordpress/android/widgets/AppReviewManager.kt

```diff
@@ -303,4 +131,18 @@ object AppReviewManager {
         AppPrefs.resetPublishedPostCount()
         AppPrefs.resetInAppReviewsNotificationCount()
     }
+
+    /**
+     * Called from various places in the app where the user has performed a non-trivial action,
+     * such as publishing a post or page. This was previously used to determine when to show our
+     * custom rating dialog to involved users but is currently unused. It is left intact, and
+     * interactions will continue to be counted, in case we want to include interactions in the
+     * future when determining whether to show the Google review dialog.
+     */
+    fun incrementInteractions(incrementInteractionTracker: AnalyticsTracker.Stat) {
+        var interactions = preferences.getInt(KEY_RATING_INTERACTIONS, 0)
```

Comment:
For my own edification: What it the rationale for relocating the `interactions` assignment from `init` to here?

Reply:
No solid rationale, other than I initially dropped interactions entirely then decided it would be worth hanging onto and put it back, this time in `incrementInteractions`.